### PR TITLE
Fixed Bug with Background Color in some Menus like 2FA

### LINKED ISSUE #18077: two factor login color issue with bright colors (theming)
### Steps to reproduce
1. set a bright color in the theming app (e.g. #ffac00)
2. enable two factor auth for a user
3. login as this user

### Expected behaviour
Login looks like this:
nc-twofactor-light-background

### Actual behaviour
Login should look like this:
nc-twofactor-light-background-readable

### Cause of the Problem:
https://github.com/nextcloud/server/blob/3860373fd42091790ac2b00039cccd22e59d0ea5/apps/theming/css/theming.scss#L181

I've deleted this line for my instance. I haven't found any problems yet.

cc @juliushaertl @MorrisJobke

### PR DESCRIPTION
**Pull request by JanDragon**
Fixes #18077 

Fixed the Background Color in some Menus like 2FA

### CODE REVIEW

**Comment by JanDragon** [comment]
There a two options. This one with set color to white or we remove this line and the system is taking the main background color.

**Comment by JanDragon** [comment]
I think this is the better way to fix it.

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
Tested and still doesn't work

Bildschirmfoto von 2019-11-27 08-48-40

**Comment by JanDragon** [comment]
Yeah I think it’s should looks like this or should it be look different?

**Comment by ChristophWurst** [comment]
Oh, I'm sorry. I think I mixed up the tickets. https://github.com/nextcloud/server/issues/14317 was the bug I was thinking about.

**Comment by JanDragon** [comment]
No problem. 🙃

**Comment by JanDragon** [comment]
No problem. 😊 

Than I think it’s ready to merge.

File: apps/theming/css/theming.scss [hunk 1]
```diff
@@ -178,7 +178,7 @@ input.primary,
 @if ($invert) {
 	#body-login {
 		.body-login-container {
```
Comment (juliushaertl) [inline apps/theming/css/theming.scss]: Please remove the whole .body-login-container block if it is not needed anymore

Comment (JanDragon) [reply apps/theming/css/theming.scss]: Done

**Comment by juliushaertl** [comment]
@JanDragon Please also squash your commits into one and add a signoff message as described here: https://probot.github.io/apps/dco/

**Comment by jancborchardt** [comment]
@JanDragon was there an issue here or is there a follow-up pull request? :) Let us know if there’s any questions or problems.

**Comment by jancborchardt** [comment]
Ah ok, new pull request seems to be at https://github.com/nextcloud/server/pull/18139

**Pull request closed without merging**